### PR TITLE
[minor_change] Add VRF, BD from "common" tenant in application_epg ex…

### LINKED ISSUE #1046: Example and clear documenation on aci_bridge_domain
It seems documenation are not clear if user created tenant wants to use BD , VRF , L3 out from "common" tenant , how they should be doing it. Can we create some additional examples which could help how "aci_application_epg" can use BD from "common" tenant , "aci_bridge_domain" VRF from "common" tenant.

### PR DESCRIPTION
**Pull request by gmicol**
Resolves #1046

### CODE REVIEW

**Review by akinross — APPROVED** [review]
LGTM

File: examples/application_epg/main.tf [hunk 1]
```diff
@@ -31,4 +42,25 @@ resource "aci_application_epg" "fooapplication_epg" {
   pref_gr_memb           = "exclude"
   prio                   = "unspecified"
   shutdown               = "no"
-}
\ No newline at end of file
+}
+
```
Comment (shrsr) [inline examples/application_epg/main.tf:46]: I suggest we add a comment here for a clear distinction from the above code snippet->

**The following depicts an example to create and associate an application EPG with the common Tenant's BD and VRF**

**Review by shrsr — CHANGES_REQUESTED** [review]

**Review by shrsr — APPROVED** [review]
LGTM

File: examples/application_epg/main.tf (hunk 1 above)
Comment (anvitha-jain) [reply examples/application_epg/main.tf:46]: Quick suggestion: we do not need 2 tenants in one main.tf 

the above file can be changed to ->

resource "aci_tenant" {}
resource "aci_vrf"
resource "aci_bridge_domain" {}
resource "aci_application_profile" {}
resource "aci_application_epg" {}

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: examples/application_epg/main.tf (hunk 1 above)
Comment (shrsr) [reply examples/application_epg/main.tf:46]: There's only one resource which is creating a tenant. Tenant "common" is queried using the data source.

**Review by anvitha-jain — APPROVED** [review]

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**